src/data/showcases.js

```diff
@@ -1216,6 +1216,15 @@ export const Showcases = [
     source: null,
     tags: ["dex", "token"],
   },
+  {
+    title: "Liqwid",
+    description:
+      "Liqwid is a non-custodial pooled lending protocol with liquid staking built on Cardano.",
+    preview: require("./showcase/liqwid.png"),
+    website: "https://app.liqwid.finance/",
+    source: null,
+    tags: ["lending", "token", "marketplace", "nftsupport"],
```

Comment:
```suggestion
    tags: ["lending", "token", "marketplace"],
```
I would be curious how the `nftsupport` option is relevant (doesn't seem so from what I see today & my current understanding of Liqwid).

Reply:
I added `nftsupport` as the Liqwid app leverages their Aquafarmers NFT collection to provide boosted staking rewards. More details can be found here: https://liqwid-finance.medium.com/liqwid-aquafarmers-nfts-mint-utility-and-protocol-owned-liquidity-30b1fae77283.

I'm also okay with removing this tag, but I would like to know if the aquafarmers would qualify for adding the tag.

Reply:
> @thenic95: Aquafarmers NFT collection ... I would like to know if the aquafarmers would qualify for adding the [`nftsupport`] tag.

(from [Liqwid-Labs > People](https://github.com/orgs/Liqwid-Labs/people)) what would you say @Banou26 @emiflake @Liqwidfinance @t4ccer to support the premise that Aquafarmers fit in this category? https://developers.cardano.org/showcase/?tags=nftsupport

Reply:
I'm guessing the aquafarmers would fit into the nftsupport tag yeah.

We also have Optim bonds coming up very soon, not 100% sure if those can be considered NFTs though.

The one I'm less sure about is the marketplace? Can a lending platform be considered one?
It seems like all lending listings also contain the marketplace tag, as per https://developers.cardano.org/showcase/?tags=lending
So that might be fine too

Reply:
@Banou26 LGTM for all @thenic95 presented tags then 😎